terra2/Makefile

```diff
@@ -0,0 +1,65 @@
+bridge_SOURCE=wormhole
+token_bridge_SOURCE=token_bridge_terra
+
+
+-include ../Makefile.help
+
+.PHONY: artifacts
+## Build contracts and copy to terra2
+artifacts:
+	cd ../cosmwasm && make artifacts
```

Comment:
nit: Always use the `$(MAKE)` variable for recursive commands.  See https://www.gnu.org/software/make/manual/html_node/MAKE-Variable.html

Reply:
nice, TIL, good suggestion. fixed.